playbooks/robusta_playbooks/pod_investigator_enricher.py

```diff
@@ -133,25 +134,29 @@ def report_pod_issue(
     # find pods with issues
     pods_with_issue = [pod for pod in pods if detect_pod_issue(pod) == issue]
     pod_names = [pod.metadata.name for pod in pods_with_issue]
```

Comment:
This is redundant
We use it only for `len`, we can use the `pods_with_issue` for that

Reply:
done